src/main/java/picard/fingerprint/FingerprintChecker.java

```diff
@@ -155,10 +158,15 @@ public void setGenotypingErrorRate(final double genotypingErrorRate) {
         this.genotypingErrorRate = genotypingErrorRate;
     }
 
+    @Deprecated //non-compliant method name.
```

Comment:
For public API deprecation, adding a javadoc would be nice.
```
/**
 * Sets the maximal difference in PL scores considered when reading PLs from a VCF.
 *
 * @deprecated since April 2020 Use {@link #setMaximalPLDifference} instead.
 */
```